draft-ietf-masque-h3-datagram.md

```diff
@@ -306,100 +383,29 @@ stream was truncated, this MUST be treated as a malformed or incomplete message.
 
 ## The Capsule-Protocol Header Field {#hdr}
 
-This document defines the "Capsule-Protocol" header field. It is an Item
-Structured Field, see {{Section 3.3 of !STRUCT-FIELD=RFC8941}}; its value MUST
-be a Boolean. Its ABNF is:
+The "Capsule-Protocol" header field is an Item Structured Field, see {{Section
+3.3 of !STRUCT-FIELD=RFC8941}}; its value MUST be a Boolean; any other value
+type MUST be handled as if the field were not present by recipients. This
+document does not define any parameters for the Capsule-Protocol header
+field value, but future documents might define parameters. Receivers MUST ignore
+unknown parameters.
 
-~~~ abnf
-Capsule-Protocol = sf-item
-~~~
+Endpoints indicate that the Capsule Protocol is in use on a data stream by
+sending a Capsule-Protocol header field with a true value. A Capsule-Protocol
+header field with a false value has the same semantics as when the header is not
+present.
 
-Endpoints indicate that the Capsule Protocol is in use on the data stream by
-sending the Capsule-Protocol header field with a value of ?1. A Capsule-Protocol
-header field with a value of ?0 has the same semantics as when the header is not
-present. Intermediaries MAY use this header field to allow processing of HTTP
-Datagrams for unknown HTTP Upgrade Tokens; note that this is only possible for
-HTTP Upgrade or Extended CONNECT.
+Intermediaries MAY use this header field to allow processing of HTTP Datagrams
+for unknown HTTP Upgrade Tokens; note that this is only possible for HTTP
+Upgrade or Extended CONNECT.
 
-The Capsule-Protocol header field MUST NOT be sent multiple times on a message.
```

Comment:
@mnot was there a reason to remove this multiplicity requirement? Can we track that as a separate issue?